[2024-04-03 14:26:27,430][0][INFO][color_logger.py:203] Logfile: /data1/xiayuyang/online_mcmt/logs/0_2024-04-03-14-26-27-430463.log
[2024-04-03 14:26:27,973][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:28,476][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:28,941][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:29,409][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:29,877][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:30,344][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:30,812][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:31,281][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:31,749][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:32,212][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:32,678][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:33,144][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:33,613][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:34,079][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:34,546][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:35,015][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:35,482][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:35,945][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:36,412][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:36,879][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:37,347][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:37,823][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:38,298][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:38,769][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:39,234][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:39,698][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:40,165][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:40,624][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:41,090][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:41,557][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:42,024][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:42,490][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:42,958][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:43,442][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:43,911][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:44,429][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:44,939][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:45,398][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:45,892][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:46,392][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:46,861][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:47,333][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:47,805][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:48,276][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:48,742][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:49,205][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:49,669][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:50,143][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:50,666][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:51,163][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:51,650][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:52,100][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:52,565][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:53,047][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:53,524][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:53,983][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:54,446][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:54,913][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:55,364][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:55,828][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:56,296][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:56,760][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:57,214][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:57,680][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:58,148][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:58,612][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:59,077][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:26:59,540][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:00,006][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:00,474][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:00,943][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:01,411][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:01,869][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:02,337][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:02,809][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:03,279][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:03,770][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:04,239][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:04,709][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:05,175][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:05,644][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:06,112][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:06,580][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:07,046][0][WARNING][generic_advisor.py:482] Cannot sample non duplicate configuration after 1000 iterations.
[2024-04-03 14:27:11,890][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.7,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:11,891][0][INFO][generic_smbo.py:295] Iter 1, objectives: [inf, inf].
[2024-04-03 14:27:14,043][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:14,044][0][INFO][generic_smbo.py:295] Iter 2, objectives: [inf, inf].
[2024-04-03 14:27:14,045][0][WARNING][generic_smbo.py:253] Evaluating duplicated configuration: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:16,245][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:16,246][0][WARNING][history.py:242] Duplicate configuration detected!
[2024-04-03 14:27:16,246][0][INFO][generic_smbo.py:295] Iter 3, objectives: [inf, inf].
[2024-04-03 14:27:16,247][0][WARNING][generic_smbo.py:253] Evaluating duplicated configuration: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:18,485][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:18,485][0][WARNING][history.py:242] Duplicate configuration detected!
[2024-04-03 14:27:18,485][0][INFO][generic_smbo.py:295] Iter 4, objectives: [inf, inf].
[2024-04-03 14:27:18,486][0][WARNING][generic_smbo.py:253] Evaluating duplicated configuration: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:20,718][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:20,719][0][WARNING][history.py:242] Duplicate configuration detected!
[2024-04-03 14:27:20,719][0][INFO][generic_smbo.py:295] Iter 5, objectives: [inf, inf].
[2024-04-03 14:27:20,719][0][WARNING][generic_smbo.py:253] Evaluating duplicated configuration: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:22,868][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:22,869][0][WARNING][history.py:242] Duplicate configuration detected!
[2024-04-03 14:27:22,869][0][INFO][generic_smbo.py:295] Iter 6, objectives: [inf, inf].
[2024-04-03 14:27:22,869][0][WARNING][generic_smbo.py:253] Evaluating duplicated configuration: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:25,061][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:25,061][0][WARNING][history.py:242] Duplicate configuration detected!
[2024-04-03 14:27:25,061][0][INFO][generic_smbo.py:295] Iter 7, objectives: [inf, inf].
[2024-04-03 14:27:25,062][0][WARNING][generic_smbo.py:253] Evaluating duplicated configuration: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:27,171][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:27,171][0][WARNING][history.py:242] Duplicate configuration detected!
[2024-04-03 14:27:27,171][0][INFO][generic_smbo.py:295] Iter 8, objectives: [inf, inf].
[2024-04-03 14:27:27,172][0][WARNING][generic_smbo.py:253] Evaluating duplicated configuration: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:29,328][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8999999999999999,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:29,329][0][WARNING][history.py:242] Duplicate configuration detected!
[2024-04-03 14:27:29,329][0][INFO][generic_smbo.py:295] Iter 9, objectives: [inf, inf].
[2024-04-03 14:27:29,330][0][WARNING][generic_advisor.py:375] No enough successful initial trials! Sample random configuration.
[2024-04-03 14:27:31,561][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.62,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:31,562][0][INFO][generic_smbo.py:295] Iter 10, objectives: [inf, inf].
[2024-04-03 14:27:31,563][0][WARNING][generic_advisor.py:375] No enough successful initial trials! Sample random configuration.
[2024-04-03 14:27:34,066][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.72,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:34,066][0][INFO][generic_smbo.py:295] Iter 11, objectives: [inf, inf].
[2024-04-03 14:27:34,067][0][WARNING][generic_advisor.py:375] No enough successful initial trials! Sample random configuration.
[2024-04-03 14:27:36,325][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.78,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:36,325][0][INFO][generic_smbo.py:295] Iter 12, objectives: [inf, inf].
[2024-04-03 14:27:36,326][0][WARNING][generic_advisor.py:375] No enough successful initial trials! Sample random configuration.
[2024-04-03 14:27:38,549][0][ERROR][generic_smbo.py:275] Exception in objective function:
Traceback (most recent call last):
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/openbox/utils/limit.py", line 62, in run_without_time_limit
    ret = obj_func(*obj_args, **obj_kwargs)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/ablation_bo.py", line 91, in multi_camera_matching
    accuracy, processing_time, section_json = profile(
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 870, in profile
    mm_data, total_detect_time, total_extract_time, total_sct_time, total_pp_time, total_match_time=run_mtmc(cams_dir=cams_dir_, weights=weights_, imgsz=imgsz_, cams_ratio=cams_ratio_, lazy_threshold=lazy_threshold_)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/S13_mcmt.py", line 540, in run_mtmc
    det_model = attempt_load(weights, map_location=device)
  File "/data1/xiayuyang/online_mcmt/detector/yolov5/models/experimental.py", line 122, in attempt_load
    model.append(torch.load(os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))), w),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 789, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1131, in _load
    result = unpickler.load()
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1101, in persistent_load
    load_tensor(dtype, nbytes, key, _maybe_decode_ascii(location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1083, in load_tensor
    wrap_storage=restore_location(storage, location),
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 1055, in restore_location
    return default_restore_location(storage, str(map_location))
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 215, in default_restore_location
    result = fn(storage, location)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/serialization.py", line 187, in _cuda_deserialize
    return obj.cuda(device)
  File "/data1/xiayuyang/anaconda3/envs/carla/lib/python3.8/site-packages/torch/_utils.py", line 80, in _cuda
    untyped_storage = torch.UntypedStorage(
torch.cuda.OutOfMemoryError: CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 9.77 GiB total capacity; 1.79 MiB already allocated; 3.06 MiB free; 2.00 MiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

config: Configuration(values={
  'fps_camera1': '1',
  'fps_camera2': '1',
  'fps_camera3': '1',
  'fps_camera4': '1',
  'lazy_threshold': 0.8,
  'model_size_camera1': 'yolov5m.pt',
  'model_size_camera2': 'yolov5m.pt',
  'model_size_camera3': 'yolov5m.pt',
  'model_size_camera4': 'yolov5m.pt',
  'resolution_camera1': '1280',
  'resolution_camera2': '1280',
  'resolution_camera3': '1280',
  'resolution_camera4': '1280',
})

[2024-04-03 14:27:38,549][0][INFO][generic_smbo.py:295] Iter 13, objectives: [inf, inf].
[2024-04-03 14:27:38,550][0][WARNING][generic_advisor.py:375] No enough successful initial trials! Sample random configuration.
